docs/section11/SHW comparison fields.yaml

```diff
@@ -0,0 +1,396 @@
+ServiceWaterHeatingDistributionSystem:
+  Data Elements:
+    id:
+      AppG 11-1 P_RMD Equals U_RMD: true
+      AppG 11-3 P_RMD Equals B_RMD: true
+    reporting_name:
+      AppG 11-1 P_RMD Equals U_RMD: false
+      AppG 11-3 P_RMD Equals B_RMD: false
+    notes:
+      AppG 11-1 P_RMD Equals U_RMD: false
+      AppG 11-3 P_RMD Equals B_RMD: false
+    design_supply_temperature:
+      AppG 11-1 P_RMD Equals U_RMD: true
+      AppG 11-3 P_RMD Equals B_RMD: true
+    design_supply_temperature_difference:
+      AppG 11-1 P_RMD Equals U_RMD: true
+      AppG 11-3 P_RMD Equals B_RMD: true
+    tanks:
+      AppG 11-1 P_RMD Equals U_RMD: true
+      AppG 11-3 P_RMD Equals B_RMD: true
+    is_central_system:
+      AppG 11-1 P_RMD Equals U_RMD: true
+      AppG 11-3 P_RMD Equals B_RMD: true
+    service_water_piping:
+      AppG 11-1 P_RMD Equals U_RMD: true
+      AppG 11-3 P_RMD Equals B_RMD: true
+    distribution_compactness:
+      AppG 11-1 P_RMD Equals U_RMD: false
+      AppG 11-3 P_RMD Equals B_RMD: false
+    control_type:
+      AppG 11-1 P_RMD Equals U_RMD: false
```

Comment:
Not following why the control type woudl not be the same

Reply:
I think it's a Title 24 element.

Reply:
we discussed on the call 5/15/2024, and we expect everything to be the same, unless there is a specific reason the two would reasonably be different.

Reply:
> we discussed on the call 5/15/2024, and we expect everything to be the same, unless there is a specific reason the two would reasonably be different.

Discussed with Christina on 5/20/2024 & she agrees with this approach